.github/ISSUE_TEMPLATE/template-graduation-application.md

```diff
@@ -187,7 +187,14 @@ Note: this section may be augmented by the completion of a Governance Review fro
 
 <!-- (Project assertion goes here) --> 
 
-- [ ] **Document the project's release process.**
+- [ ] **Document the project's release process and guidelines publicly in a RELEASES.md or equivalent file that defines:** 
+
+  - [ ] release expectations (scheduled or based on feature implementation)
+  - [ ] tagging as stable, unstable, and security related releases
+  - [ ] information on branch and tag strategies
+  - [ ] branch and platform support and length of support
+  - [ ] artifacts included in the release.
```

Comment:
I think that there are the only checkboxes with text in this file that don't start with a capital letter. We should probably make them consistent with the rest of the file.

Reply:
```suggestion
  - [ ] Release expectations (scheduled or based on feature implementation)
  - [ ] Tagging as stable, unstable, and security related releases
  - [ ] Information on branch and tag strategies
  - [ ] Branch and platform support and length of support
  - [ ] Artifacts included in the release.
```